models/gc_picai_baseline/meta.json

```diff
@@ -0,0 +1,138 @@
+{
+  "id": "c5f886fb-9f54-4555-a954-da02b22d6d3f",
+  "name": "picai_baseline",
+  "title": "PI-CAI challenge baseline",
+  "summary": {
+    "description": "The PI-CAI challenge is to validate modern AI algorithms at clinically significant prostate cancer (csPCa) detection and diagnosis. This model algorithm provides the baseline for the challenge.",
+    "inputs": [
+      {
+        "label": "Prostate biparametric MRI",
+        "description": "Prostate biparametric MRI exam",
+        "format": "DICOM",
+        "modality": "MR",
+        "bodypartexamined": "Prostate",
+        "slicethickness": "",
+        "non-contrast": false,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "type": "Prediction",
+        "valueType": "Probability",
+        "label": "Prostate cancer probability",
+        "description": "Case-level likelihood of harboring clinically significant prostate cancer, in range [0,1]",
+        "classes": []
+      },
+      {
+        "type": "Prediction",
+        "valueType": "Probability map",
+        "label": "Transverse cancer detection map",
+        "description": "Detection map of clinically significant prostate cancer lesions in 3D, where each voxel represents a floating point in range [0,1]",
+        "classes": []
+      }
+    ],
+    "model": {
+      "architecture": "3d fullres nnUNet",
+      "training": "supervised",
+      "cmpapproach": "3D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 1200
+      },
+      "evaluation": {
+        "vol_samples": 300
+      },
+      "public": false,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "PI-CAI challenge baseline",
+    "version": "v2.1.1",
+    "devteam": "Diagnostic Image Analysis Group, Radboud University Medical Center, Nijmegen, The Netherlands",
+    "type": "Prediction",
+    "date": {
+      "weights": "2022-06-22",
+      "code": "2022-09-05",
+      "pub": ""
+    },
+    "cite": "",
+    "license": {
+      "code": "Apache 2.0",
+      "weights": "Apache 2.0"
+    },
+    "publications": [],
+    "github": "https://github.com/DIAGNijmegen/picai_nnunet_semi_supervised_gc_algorithm",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "Prediction of the likelihood of harboring clinically significant prostate cancer (csPCa) in prostate biparametric MRI exams.",
```

Comment:
This is *what* the model does, in this section, we should describe *how* to use this model to get there.
E.g., if you are interested in ... you need ... and ... image data prepared in a folder ... to run the model ...
This is especially useful, as for multi-input models the run-instructions are less simple than for those starting with a DicomImporter.

Reply:
You can use an adapted version of the description from GC:

This algorithm is a deep learning-based detection/diagnosis model, which ensembles 5 independent nnU-Net models (5-fold cross-validation). To predict the likelihood of harboring clinically significant prostate cancer (csPCa), the transversal T2-weighted, apparent diffusion coefficient (ADC) and high b-value diffusion maps are required. The input sequences should be co-registered or aligned reasonably well and the prostate gland should be  localized within a volume of 460 cm³ from the centre coordinate. To train these models, a total of 1500 prostate biparametric MRI (bpMRI) scans paired with human-annotated or AI-derived ISUP ≥ 2 delineations were used.

Reply:
For now, we will use Joeran's suggested description. However, it seems that @LennyN95 also implies a more hands-on instruction for how to prepare and run the actual model on patient data. Is that correct? Does that information have a place in the model card or is the description as described by @joeranbosma sufficient?

Reply:
The description of the required files itself is fine. You could append to the intended use the folder structure as this is arbitrary to the model. I will implement some more generalizable ways in the future based on all the multi-input submissions like this one then.

Reply:
I have appended the intended folder structure to the intended use section.